driver/p11_driver.mli

```diff
@@ -119,6 +120,9 @@ end
 type t = (module S)
 
 val initialize : t -> unit
+(** Perform a c_Initialize call with NSS-style initialization parameters as described
+    at https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS/PKCS11/Module_Specs *)
+val initialize_nss : t -> params: Pkcs11.Nss_initialize_arg.u -> unit
```

Comment:
To ensure the docstring is associated with that function (notably by documentation generators), I think it would make sense to surround "docstring + val initialize_nss : t -> ..." with spaces.  Otherwise, I think the docstring might be associated with `initialize` instead.